main.nf

```diff
@@ -344,15 +420,15 @@ if (params.bam) {
     maxRetries 1
 
     input:
-    ksize
-    molecule
-    log2_sketch_size
+    each ksize from ksizes
+    each molecule from molecules
```

Comment:
this was intentional! I don't wanna compute and process all ksizes, molecule types for all permutations for every bam. I want to do it once on the first molecule and just do sourmash compute on the fastas formed later, please change it back!